app/internal/wm/os_js.go

```diff
@@ -583,6 +590,29 @@ func (w *window) windowMode(mode WindowMode) {
 	}
 }
 
+func (w *window) navigationColor(c color.NRGBA) {
+	if c.A != 255 {
+		// Browser doesn't support alpha.
+		c = color.NRGBA{
+			R: c.A*c.R + (255 - c.A),
+			G: c.A*c.G + (255 - c.A),
+			B: c.A*c.B + (255 - c.A),
```

Comment:
This doesn't seem right. For example, if c.A = 254, c.R = 255, the result is much larger than 255. Except that it isn't, because the result overflows uint8.